tests/lit-tests/1631.ispc

```diff
@@ -0,0 +1,44 @@
+//; RUN: %{ispc} %s -o %t.o --nowrap --target=avx2-i32x8 2>&1 | FileCheck %s
+//; CHECK: Warning: Converting from const pointer type "const uniform float * uniform" to pointer type "uniform float *
+//uniform" for function call argument discards const qualifier.
```

Comment:
You can't split this to extra line, it's basically ignored.

Reply:
That was a mistake. The first line should be enough to check. Took out the 2nd part.